CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 v2.3.0
 
+* Added support for polymorphic models via `django-polymorphic`
```

Comment:
Well, so actually this is only tested against `django-polymorphic`, which is the most popular and maintained polymorphic library for Django. But the implemented polymorphic support is library-agnostic and should work with libraries like `django-polymodels`, `django-typed-models` or even Django's `contrib.GenericRelation`...

I'm not sure how to deal with it, maybe an explicit warning, I'm not convinced to add more tests with more libraries... Any thoughts?